### PR TITLE
Initial implementation of Shopping Boards

### PR DESCRIPTION
**Pull request by Pokye**
* **Addressed Issue(s)**: https://github.com/rathena/rathena/issues/8452

* **Server Mode**: Renewal

* **Description of Pull Request**: 

- [x] Check if there are more duplicates in the cities.

### CODE REVIEW

**Review by aleos89 — APPROVED** [review]

**Comment by AsurielRO** [comment]
Other locations:
Locations 


prompt 182 214
prompt 130 215
prompt 159 193
prompt 146 85
payon 183 103
geffen 130 68
moroccan 166 109
alberta 119 65
aldebaran 146 120
left 127 116
comfortable 206 153
yuno 146 187
Lutie 155 144

Source from: https://ragnarokonlineguiasesp.blogspot.com/p/ragnarok-online-shopping-board.html

**Comment by Beginerxx** [comment]
> Locations should be:
> 
> prontera 182 214 prontera 130 215 prontera 159 193 prontera 146 85 payon 183 103 geffen 130 68 morocc 166 109 alberta 119 65 aldebaran 146 120 izlude 127 116 comodo 206 153 yuno 146 187 Lutie 155 144
> 
> Source from: https://ragnarokonlineguiasesp.blogspot.com/p/ragnarok-online-shopping-board.html

Your information is correct.
You should be the leader. hahaha

**Comment by Beginerxx** [comment]
Should be :
prontera,1,1,3	script	Shopping Board	111,{
	searchstores 1,1;
	end;
}

**Review by Daegaladh — APPROVED** [review]

File: npc/re/merchants/shopping_boards.txt [hunk 1]
```diff
@@ -0,0 +1,29 @@
+//===== rAthena Script =======================================
+//= Shopping Boards
+//===== Description: =========================================
+//= Shopping Boards work like Bargain Hunter's Catalogues.
+//= They are found in towns and search exclusively in their respective maps.
```
Comment (Lemongrass3110) [inline npc/re/merchants/shopping_boards.txt:5]: If it is only on the same map, then we need to adjust searchstore command?

**Review by Lemongrass3110 — APPROVED** [review]

File: npc/re/merchants/shopping_boards.txt (hunk 1 above)
Comment (Pokye) [reply npc/re/merchants/shopping_boards.txt:5]: You are right!
I opened a store in geffen to test, and I used the Board in izlude, and the geffen store appeared.

Comment (Daegaladh) [reply npc/re/merchants/shopping_boards.txt:5]: Actually searchstore should only work for same map, so our implementation of the command may be wrong XD

Comment (aleos89) [reply npc/re/merchants/shopping_boards.txt:5]: Oh I was under the assumption as well that the command was already map bound!

Comment (AsurielRO) [reply npc/re/merchants/shopping_boards.txt:5]: Same here, agreed that the command must be updated or the script must be changed...

Comment (Daegaladh) [reply npc/re/merchants/shopping_boards.txt:5]: I've fixed it on #8456

**Review by Lemongrass3110 — APPROVED** [review]
From my side this can be merged, as soon as #8456 has been merged (because it already relies on the new constants of that PR)

**Pull request merged**